src/server.c

```diff
@@ -6275,7 +6275,7 @@ sds getVersion(void) {
 }
 
 void usage(void) {
-    fprintf(stderr,"Usage: ./redis-server [/path/to/redis.conf] [options] [-]\n");
+    fprintf(stderr,"Usage: ./redis-server [/path/to/valkey.conf] [options] [-]\n");
```

Comment:
Here, all redis-server should be updated to valkey-server

Reply:
Will address this in a follow up.